### PR TITLE
ticket 63- change in the print link and region added in the info file

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: sites/all/themes/custom/adb/sass/utilities/_mixins.scss [hunk 1]
```diff
@@ -100,3 +100,9 @@
   overflow: hidden;
   outline: 0;
 }
+
+
+@mixin flex-direction($dir) {
+  -webkit-flex-direction: $dir;
+  flex-direction: $dir; 
+}
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/utilities/_mixins.scss]: There should be a newline

File: sites/all/themes/custom/adb/templates/field-collection-item--field_impact.tpl.php [hunk 2]
```diff
@@ -28,14 +28,14 @@
...
-    <div class="impact-value-unit-wrapper">
-      <?php print render($content['field_value']); ?>
-      <?php print render($content['field_unit']); ?>
-    </div>
-    <div class="impact-description-wrapper">
-      <?php print render($content['field_description']); ?>
-    </div>
-  </div>
-</div>
+	<div class="<?php print $classes; ?> clearfix impact-list"<?php print $attributes; ?>>
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/templates/field-collection-item--field_impact.tpl.php]: Correct this indentation.

File: sites/all/themes/custom/adb/templates/page.tpl.php [hunk 3]
```diff
@@ -160,11 +163,11 @@
             <?php if ($title || $primary_local_tasks || $secondary_local_tasks || $action_links = render($action_links)): ?>
               <header<?php print $content_header_attributes; ?>>
 
-                <?php if ($title): ?>
+                <!--<?php if ($title): ?>
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/templates/page.tpl.php]: Remove commented code.

File: sites/all/modules/features/adb_case_study/adb_case_study.features.field_instance.inc [hunk 4]
```diff
@@ -333,6 +333,45 @@ function adb_case_study_field_default_field_instances() {
     ),
   );
 
+
+  // Exported field_instance:
+  // 'field_collection_item-field_impact-field_description_body'.
+  $field_instances['field_collection_item-field_impact-field_description_body'] = array(
```
Comment (subhojit777) [inline sites/all/modules/features/adb_case_study/adb_case_study.features.field_instance.inc]: @surbhi90 @shamsher327 has already removed this from database. Remove this field from your local, and update the feature. This feature code shouldn't come in this PR.

File: sites/all/themes/custom/adb/sass/component/_sidebar.scss [hunk 5]
```diff
@@ -2,3 +2,16 @@
...
+    float: none;
+    margin: 5px 0 15px;
+    &:first-child {
+      margin: 0 0 15px;
+    }
+  }
+  a {
+    font-size: ($base-font)+2;
+  }
+}
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/component/_sidebar.scss]: There should be a newline.

File: sites/all/themes/custom/adb/sass/layout/_layout.scss [hunk 6]
```diff
@@ -34,4 +34,26 @@ header {
...
-}
\ No newline at end of file
+}
+
+
+.sidebar-first-pages {
+  @extend %flex-row;
+  @extend %wrap;
+  @include flex-direction(row-reverse);
+  /*justify-content: flex-end;*/
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/layout/_layout.scss]: @surbhi90 If you don't need this then remove this.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 7]
```diff
@@ -203,30 +224,27 @@
 	@extend %hide;
   }
   ul {
-	padding: 0 20px 0 120px;
-	margin: 0;
-	list-style: none;
+  	padding: 0 20px 0 120px;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 8]
```diff
@@ -203,30 +224,27 @@
...
+  	margin: 0;
+  	list-style: none;
   }
   li{
-	width: 47%;
-	display: inline-block;
-	vertical-align: top;
-	margin: 15px 1%;
+  	width: 47%;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 9]
```diff
@@ -203,30 +224,27 @@
...
+  	vertical-align: top;
+  	margin: 15px 1%;
   }
   a {
-	color: $link;
-	font-size: ($base-font)+10;
-	&.active {
-	  color: $link;
-	}
+  	color: $link;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 10]
```diff
@@ -203,30 +224,27 @@
...
+  	font-size: ($base-font)+10;
   }
   .fa {
-	color: $grey;
-	font-size: ($base-font)+16;
-	position: absolute;
-	top: -11px;
-	left: 50%;
-	@include transform (translateX(-50%));
+  	color: $grey;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 7 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: It is already coming fine in my file

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 8 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: It is already coming fine you have to check the indentation from the li and not from the left start as it is in hierarchy

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 9 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: It is already coming fine you have to check the indentation from the li and not from the left start as it is in hierarchy

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 10 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: It is already coming fine you have to check the indentation from the li and not from the left start as it is in hierarchy

File: sites/all/themes/custom/adb/sass/utilities/_mixins.scss (hunk 1 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/utilities/_mixins.scss]: done

File: sites/all/themes/custom/adb/sass/component/_sidebar.scss (hunk 5 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/component/_sidebar.scss]: done

**Pull request merged**